deployments/bridges/eth-poa-sub/docker-compose.yml

```diff
@@ -0,0 +1,101 @@
+# This Compose file should be built using the Rialto and Eth-PoA node
+# compose files. Otherwise it won't work.
+
+version: '3.5'
+services:
+  # We override these nodes to make sure we have the correct chain config for this network.
+  poa-node-arthur: &poa-node
+    volumes:
+      - ./bridges/eth-poa-sub/poa-config:/config
+  poa-node-bertha:
+      <<: *poa-node
+  poa-node-carlos:
+      <<: *poa-node
+
+  # We provide an override for this particular node since this is a public facing
+  # node which we use to connect from things like Polkadot JS Apps.
+  rialto-node-charlie:
+    environment:
+      VIRTUAL_HOST: rialto.bridges.test-installations.parity.io,wss.rialto.brucke.link
+      VIRTUAL_PORT: 9944
+      LETSENCRYPT_HOST: rialto.bridges.test-installations.parity.io,wss.rialto.brucke.link
+      LETSENCRYPT_EMAIL: admin@parity.io
+
+  relay-eth2sub:
+    image: paritytech/ethereum-poa-relay
+    entrypoint: /config/relay-eth2sub-entrypoint.sh
+    volumes:
+      - ./bridges/eth-poa-sub/scripts:/config
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
+    ports:
+      - "9616:9616"
+    depends_on: &all-nodes
+      - poa-node-arthur
+      - poa-node-bertha
+      - poa-node-carlos
+      - rialto-node-alice
+      - rialto-node-bob
+      - rialto-node-charlie
+      - rialto-node-dave
+      - rialto-node-eve
+
+  poa-exchange-tx-generator:
+    image: paritytech/ethereum-poa-relay
+    entrypoint: /config/poa-exchange-tx-generator-entrypoint.sh
+    volumes:
+      - ./bridges/eth-poa-sub/scripts:/config
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
+      EXCHANGE_GEN_MIN_AMOUNT_FINNEY: ${EXCHANGE_GEN_MIN_AMOUNT_FINNEY:-1}
+      EXCHANGE_GEN_MAX_AMOUNT_FINNEY: ${EXCHANGE_GEN_MAX_AMOUNT_FINNEY:-100000}
+      EXCHANGE_GEN_MAX_SUBMIT_DELAY_S: ${EXCHANGE_GEN_MAX_SUBMIT_DELAY_S:-60}
+    depends_on: *all-nodes
+
+  relay-eth-exchange-sub:
+    image: paritytech/ethereum-poa-relay
+    entrypoint: /config/relay-eth-exchange-sub-entrypoint.sh
+    volumes:
+      - ./bridges/eth-poa-sub/scripts:/config
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
+    ports:
+      - "9716:9616"
+    depends_on: *all-nodes
+
+  relay-sub2eth:
+    image: paritytech/ethereum-poa-relay
+    entrypoint: /config/relay-sub2eth-entrypoint.sh
+    volumes:
+      - ./bridges/eth-poa-sub/scripts:/config
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
+    ports:
+      - "9618:9616"
+    depends_on: *all-nodes
+
+  front-end:
+    build:
+      context: .
+      dockerfile: ./bridges/eth-poa-sub/Front-end.Dockerfile
+      args:
+        SUBSTRATE_PROVIDER: ${UI_SUBSTRATE_PROVIDER:-ws://localhost:9944}
+        ETHEREUM_PROVIDER: ${UI_ETHEREUM_PROVIDER:-http://localhost:8545}
+        EXPECTED_ETHEREUM_NETWORK_ID: ${UI_EXPECTED_ETHEREUM_NETWORK_ID:-105}
+    ports:
+      - "8080:80"
+
+  # Note: These are being overridden from the top level `monitoring` compose file.
+  prometheus-metrics:
```

Comment:
I now see why monitoring is required. Can we just provide some dummy image (maybe `scratch`?) to make sure this service is complete that is later on overwritten by including `monitoring.yml`?

Reply:
We can't do this since `scratch` isn't actually an image. We _could_ make a dummy image `FROM scratch` and use that, but I'd rather just make the monitoring mandatory at that point

Reply:
We can just use a well established image like:
```
image: alpine:3.6
```

Let's create an issue for this if you refuse to fix this in that PR.

Reply:
Logged here: #484